serverFiles/models/comments.models.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../../db/connection');
+
+function addCommentByArticleId(article_id, commentData) {
+  const regexPattern = /\d/g;
+  const isNumber = regexPattern.test(article_id);
```

Comment:
This is a cool solution to check the type of the ID, PSQL should reject the promise with an error automaticaly if we try post a comment to a bad id however I don't see the harm in filtering this out before it gets to the database.